src/server/config/swagger.json

```diff
@@ -1197,10 +1188,9 @@
         "eta": "2019-09-10 21:20:09",
         "forwarddraft": 8.596,
         "aftdraft": 8.785,
-        "optimisation_type": 9.852,
-        "depart_from": 8,
-        "arrive_at": 7,
-        "sailtime": 98.879,
+        "optimisation_type": 3.8,

```

Comment:
Optimisation type should not be a number but either:

`least_cost_arrival`, `fixed_eta`, or `earliest_arrival`

Reply:
changed  "optimisation_type" to "fixed_eta"